walkthrough-1A/README.md

```diff
@@ -0,0 +1,60 @@
+# Prerequisites
+
+- The installation has already been performed on an OpenShift cluster as per the installation document https://github.com/integr8ly/installation/blob/master/README.md.
+
+# Setting up the Enmasse Address
+Open the `Enmasse Console` and go to the Addresses tab. Create a new address, 
+name it `work-queue/requests`, select `queue` as the type and click `Next`. 
```

Comment:
Just a note that creating a new address means is clicking the `Create` button at the top of the page (centered, if that matters).